src/css/components/_spinner.scss

```diff
@@ -1,16 +1,18 @@
+// stylelint-disable declaration-no-important
+
 @keyframes container-rotate {
-  to { transform: rotate(360deg) }
+  to { transform: rotate(360deg); }
 }
 
 @keyframes fill-unfill-rotate {
-  12.5% { transform: rotate(135deg);  } /* 0.5 * ARCSIZE */
-  25%   { transform: rotate(270deg);  } /* 1   * ARCSIZE */
-  37.5% { transform: rotate(405deg);  } /* 1.5 * ARCSIZE */
-  50%   { transform: rotate(540deg);  } /* 2   * ARCSIZE */
-  62.5% { transform: rotate(675deg);  } /* 2.5 * ARCSIZE */
-  75%   { transform: rotate(810deg);  } /* 3   * ARCSIZE */
-  87.5% { transform: rotate(945deg);  } /* 3.5 * ARCSIZE */
-  to    { transform: rotate(1080deg); } /* 4   * ARCSIZE */
+  12.5% { transform: rotate(135deg); } // 0.5 * ARCSIZE
```

Comment:
Changed the comments to `//` so that they are not output in the CSS. With the `compressed` style option this doesn't change anything, though.